<launch>
  <node pkg="valve_tracker_triton" type="valve_trainer_node" name="valve_trainer" output="screen">
    <param name="num_hue_bins" value="16"/>
    <param name="num_sat_bins" value="16"/>
    <param name="num_val_bins" value="1"/>

    <remap from="stereo" to="stereo_down"/>
    <remap from="image" to="image_rect_color"/>
  </node>
</launch>
